Tools/ardupilotwaf/boards.py

```diff
@@ -286,15 +286,15 @@ def configure_env(self, cfg, env):
 
         if cfg.env.DEBUG:
             env.CFLAGS += [
-                '-g',
+                '-g3',
```

Comment:
check works on clang? or enable only on gcc

Reply:
--debug build added to CI check.